components/BlurryPoster/BlurryPoster.tsx

```diff
@@ -241,7 +242,7 @@ export const BlurryPoster = ({
                     alt={alt || ''}
                     width={type === 'hero' && !isTwoCol ? 1800 : 750}
                     height={type === 'hero' && !isTwoCol ? 900 : 1000}
-                    aria-describedby={hasCaption ? 'story-hero-caption' : undefined}
+                    aria-describedby={hasCaption && !!alt ? 'story-hero-caption' : undefined}
```

Comment:
Don't add aria-describedby to foreground image unless it has an alt (otherwise it's decorative and shouldn't have caption associated with it)